examples/graphql-client-error-handling/graphql_client_error_handling.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/graphql;
+import ballerina/io;
+
+type ProfileResponse record {|
+    *graphql:GenericResponseWithErrors;
+    record {|Person profile;|} data;
+|};
+
+type Person record {|
+    string name;
+    int age;
+|};
+
+public function main() returns error? {
+    do {
+        graphql:Client graphqlClient = check new ("localhost:9090/graphql");
+        // This is a malformed GraphQL document.
+        string document = "mutation { updateName(name: 1) { name, age } }";
+        ProfileResponse response = check graphqlClient->execute(document);
+        io:println(response);
+    } on fail graphql:ClientError err {
+        handleErrors(err);
+    }
+}
+
+function handleErrors(graphql:ClientError clientError) {
+    if clientError is graphql:PayloadBindingError {
+        // This error represents a client-side data binding error. This error occurs due to the
+        // assigned variable type and the value obtained from the wire having a mismatching 
+        // shape.
+        io:println("PayloadBindingError: ", clientError.cause());
```

Comment:
Cause can be `nil`, right? Better to use the error message here.

```suggestion
        io:println("PayloadBindingError: ", clientError.message());
```

Reply:
Updated